ckanext/s3filestore/cli_commands.py

```diff
@@ -112,12 +112,14 @@ def upload_single(self, id):
         print('{0} resources matched on the database'.format(
             len(resource_ids_and_names.keys())))
 
-        BASE_PATH = config.get('ckan.storage_path', '/var/lib/ckan/default/resources')
+        BASE_PATH = os.path.join(config.get('ckan.storage_path', '/var/lib/ckan/default/'), 'resources')
         resource_ids_and_paths = {}
         for resource_id in resource_ids_and_names.keys():
-            path = '{}/{}/{}/{}'.format(BASE_PATH, resource_id[0:2], resource_id[3:5], resource[6:])
+            path = '{}/{}/{}/{}'.format(BASE_PATH, resource_id[0:3], resource_id[3:6], resource_id[6:])
```

Comment:
yep, big issue,  0:2 should have been 0:3 ...

https://github.com/ckan/ckan/blob/14bdff2453d3f78c17e26dfb1ba8fa8ac4df7e91/ckan/lib/uploader.py#L292

Reply:
This was for the 'single resource' task, which wasn't what we used when migrating (but was quite helpful for debugging this, once it was fixed up).